project/scans/components/form/CreateScanForm.vue

```diff
@@ -186,6 +192,32 @@ export default defineComponent({
       finished: false,
       assetPlatformType: AssetEnum.ANDROID_PLAYSTORE,
```

Comment:
Since you switched the order, the default should be `AssetEnum.WEB_APP`, and then in `resetForm` you should do `this.assetPlatformType = AssetEnum.WEB_APP`